### PR TITLE
Fix build and lint workflows 

### PR DESCRIPTION
**Pull request by ArmanA30**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

Fixes #(issue)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Open an issue for bug report or feature request and mention it to this PR.
- [ ] The commit message follows our guidelines: <https://github.com/AliMD/alwatr/blob/main/CONTRIBUTING.md#commit-message-format>
- [ ] All methods have proper description in `jsdocs` format with `@example` section.
- [ ] Docs have been added / updated (for bug fixes / features).
- [ ] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by ArmanA30** [comment]
@AliMD  OK?

File: package.json [hunk 1]
```diff
@@ -19,13 +19,12 @@
     "pp": "npm-run-all --sequential clean build ver-patch publish",
     "lint": "npm-run-all --sequential lint:*",
     "lint:ts": "eslint **/*.ts",
-    "build": "npm-run-all --sequential lint:* build:*",
+    "build": "npm-run-all --sequential build:ts build:es",
```
Comment (AliMD) [inline package.json:22]: ما اینجا build:es نداریم
خودت ببین تو اسم اسکریپت ها

**Review by AliMD — CHANGES_REQUESTED** [review]
Tnank you 👍🏻

File: package.json (hunk 1 above)
Comment (ArmanA30) [reply package.json:22]: خودم شک کردم 
ولی چون گفته بودید بر اساس alwatr-pwa این تغییرات انجام بشه...!
 الان alwatr-pwa هم همین اشکال رو داره...؟
 https://github.com/AliMD/alwatr-pwa/blob/next/package.json

Comment (AliMD) [reply package.json:22]: خودم انجام میدم

**Review by AliMD — APPROVED** [review]

**Pull request merged**